Documentation/network/servicemesh/mutual-authentication/installation.rst

```diff
@@ -9,6 +9,25 @@ Installation
 
 .. tabs::
 
+    .. group-tab:: Cilium CLI
+
+        .. include:: ../../../installation/cli-download.rst
+
+        Mutual authentication and associated SPIRE server can be enabled with the following command.
+        Note this requires the Cilium CLI Helm mode (version 0.15 or later).
```

Comment:
Use active voice, specific is better than general, punctuation nits

I'm not attached to "its" re:SPIRE server, feel free to replace with another article like "an"
```suggestion
        You can enable mutual authentication and its associated SPIRE server with the following command.
        This command requires the Cilium CLI Helm mode version 0.15 or later.
```